<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventWidget</class>
 <widget class="QWidget" name="EventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>471</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Main informations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="eventTypeLabel">
        <property name="text">
         <string>Type</string>
        </property>
        <property name="buddy">
         <cstring>eventTypeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="eventTypeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="voteLabel">
        <property name="text">
         <string>Vote</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" colspan="3">
       <widget class="QxtStars" name="starsSlider" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="startLabel">
        <property name="text">
         <string>Start time</string>
        </property>
        <property name="buddy">
         <cstring>startTimeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="startTimeEdit">
        <property name="displayFormat">
         <string>h:mm AP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="9">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>nameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="2" column="5" colspan="6">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Long description</string>
        </property>
        <property name="buddy">
         <cstring>descriptionLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="10">
       <widget class="QLineEdit" name="descriptionLineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="distanceLabel">
        <property name="text">
         <string>Distance</string>
        </property>
        <property name="buddy">
         <cstring>distanceDoubleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="distanceDoubleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix">
         <string> km</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="7">
       <widget class="QLabel" name="durationLabel">
        <property name="text">
         <string>Duration</string>
        </property>
        <property name="buddy">
         <cstring>durationTimeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="8" colspan="3">
       <widget class="QTimeEdit" name="durationTimeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>h:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="paceLabel">
        <property name="text">
         <string>Computed pace</string>
        </property>
        <property name="buddy">
         <cstring>paceLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="10">
       <widget class="QLineEdit" name="paceLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>232</red>
              <green>232</green>
              <blue>232</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>232</red>
              <green>232</green>
              <blue>232</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>232</red>
              <green>232</green>
              <blue>232</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="shoeLabel">
        <property name="text">
         <string>Shoes</string>
        </property>
        <property name="buddy">
         <cstring>shoeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="10">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="shoeComboBox"/>
        </item>
        <item>
         <widget class="QToolButton" name="shoeToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="paceLabel_2">
        <property name="text">
         <string>Notes</string>
        </property>
        <property name="buddy">
         <cstring>notesPlainTextEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="10">
       <widget class="QPlainTextEdit" name="notesPlainTextEdit"/>
      </item>
      <item row="0" column="10">
       <widget class="QPushButton" name="votePushButton">
        <property name="text">
         <string>Vote details ...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="eventTypeToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Additional informations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="intervalsPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Interval's details ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runnerInfoPushButton">
          <property name="text">
           <string>Runner informations ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="weatherInfoPushButton">
          <property name="text">
           <string>Weather informations ...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="resetPushButton">
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/application.qrc">
         <normaloff>:/images/reset</normaloff>:/images/reset</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>338</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/application.qrc">
         <normaloff>:/images/save</normaloff>:/images/save</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/application.qrc">
         <normaloff>:/images/cancel</normaloff>:/images/cancel</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxtStars</class>
   <extends>QWidget</extends>
   <header>widgets/qxtstars/qxtstars.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/application.qrc"/>
 </resources>
 <connections/>
</ui>
